### DISCUSSION TITLE
Welcome to moviepy Discussions!

### DISCUSSION THREAD

**Discussion by Zulko** [Zulko/moviepy#1387] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vsingh9076** [comment]
What video formats are supported by moviepy ?

**Reply by mondeja** [reply]
MoviePy uses FFmpeg under the hood, so you should be able to use all formats supported by your FFmpeg installation.

**Reply by tburrows13** [reply]
But if the extension is not in this dictionary then you'll need to specify your own codec: https://github.com/Zulko/moviepy/blob/master/moviepy/tools.py#L129-L139
